src/pocketmine/Player.php

```diff
@@ -234,6 +234,9 @@ public static function isValidUserName(?string $name) : bool{
 	/** @var string */
 	protected $xuid = "";
 
+	/** @var int */
```

Comment:
Use `$deviceOs` instead of `$deviceOS`

Reply:
In my opinion this isn't necessary. `deviceOs` looks odd and unintuitive imho.

Reply:
But it better fits how variables are called in PMMP.

Reply:
`$deviceOS` is better since the function is named `getDeviceOS()`